cypress/integration/pipeline.plugin.multipleendpoints.spec.ts

```diff
@@ -116,16 +116,20 @@ describe('Pipelines with plugins having more than one endpoints', () => {
       emitter.emit(input);
     }`;
     cy.add_node_to_canvas(fileSourceId);
-    cy.open_transform_panel();
+    cy.toggle_source_panel();
+    cy.toggle_transform_panel();
```

Comment:
Because of the new product requirement, now the left panel is able to open multiple accordions at the same time. So e2e tests need to explicitly close unwanted accordions otherwise bottom plugins cannot be clicked